src/Take.Blip.Builder/AttendanceRedirectHandler.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Lime.Protocol;
+using Take.Blip.Builder.Hosting;
+using Take.Blip.Client.Extensions.HelpDesk;
+using Takenet.Iris.Messaging.Resources;
+
+namespace Take.Blip.Builder
+{
+    public class AttendanceRedirectHandler : IAttendanceRedirectHandler
+    {
+
+        public static string ATTENDANCE_REDIRECT_KEY = "attendanceRedirect";
```

Comment:
This keys are being used in another class? Make them private string only if not.

Reply:
its beeing used on tests